static/app/components/feedback/widget/useFeedbackForm.tsx

```diff
@@ -10,20 +10,22 @@ export function useFeedbackForm(props: UseFeedbackOptions) {
   const formRef = useRef<FeedbackDialog | null>(null);
   const {feedback, options} = useFeedback(props);
 
-  const openForm = useCallback(async () => {
-    if (!feedback) {
-      return;
-    }
+  const openForm = useCallback(
```

Comment:
Mostly whitespace changes. The big difference is that it now accepts option overrides in `openForm()` and removes the old form when another one is opened.